tests/extract-rpi-bootdata.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+VM_IMAGE="$1"
+WORK_DIR="${WORK_DIR:-./rpi-work}"
+KERN_DIR="${KERN_DIR:-./rpi-kernel}"
+
+bail_noremove() {
+  echo "$1" >&2
+  exit 1
+}
+
+cleanup() {
+  if mountpoint "$WORK_DIR" 2>/dev/null >/dev/null; then
```

Comment:
that's quite unusual syntax for bash scripts, I'd prefer `>/dev/null 2>&1` or `&>/dev/null` instead (same below)